Library/Homebrew/test/support/helper/test_case.rb

```diff
@@ -12,6 +12,20 @@ class TestCase < ::Minitest::Test
     TEST_SHA1   = "deadbeefdeadbeefdeadbeefdeadbeefdeadbeef".freeze
     TEST_SHA256 = "deadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeef".freeze
 
+    def before_setup
```

Comment:
Can we use `setup` instead of `before_setup`?

Minitest has this to say, which is why I went to great lengths to avoid them:

> These are meant for library writers, NOT for regular test authors.